packages/manager/modules/pci/src/projects/project/ai/apps/dashboard/dashboard.html

```diff
@@ -1,42 +1,62 @@
-<oui-header
-    data-heading="{{ $ctrl.app.spec.name }}"
-    description="{{ $ctrl.app.id }}"
->
-    <oui-header-tabs>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.generalInformationLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.generalInformationLink"
-            ><span data-translate="pci_apps_dashboard_info_tab_title"></span>
-        </oui-header-tabs-item>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.attachDataLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.attachDataLink"
-            ><span
-                data-translate="pci_apps_dashboard_attach_data_tab_title"
-            ></span>
-        </oui-header-tabs-item>
-    </oui-header-tabs>
-    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
-        <oui-guide-menu-item
-            data-ng-repeat="guide in $ctrl.guideUrl.ai_machine_learning track by $index"
-            data-href="{{:: guide.url }}"
-            data-on-click="$ctrl.trackClick($ctrl.guideTrackingSectionTags.ai_apps + guide.tracking)"
-            data-external
+<header class="oui-header">
+    <div class="oui-header__container">
+        <div class="oui-header__content">
+            <h1
+                class="oui-header__title"
+                data-ng-bind="$ctrl.app.spec.name"
+            ></h1>
+            <h2
+                class="oui-header__description"
+                data-ng-bind="$ctrl.app.id"
+            ></h2>
+        </div>
+        <oui-guide-menu
+            data-text="{{:: 'pci_project_guides_header' | translate }}"
         >
-            <span
-                data-translate="{{:: 'pci_project_guides_header_' + guide.key }}"
-            ></span>
-        </oui-guide-menu-item>
-    </oui-guide-menu>
-</oui-header>
-<oui-message data-type="info" dismissable>
-    <span data-translate="pci_apps_dashboard_banner_user_and_roles_info"></span>
-    <p class="m-0">
-        <a
-            rel="noopener"
-            data-translate="pci_apps_dashboard_banner_user_and_roles_info_link"
-            data-ng-click="$ctrl.goToGenerateToken()"
-        ></a>
-    </p>
-</oui-message>
+            <oui-guide-menu-item
+                data-ng-repeat="guide in $ctrl.guideUrl.ai_machine_learning track by $index"
+                data-href="{{:: guide.url }}"
+                data-on-click="$ctrl.trackClick($ctrl.guideTrackingSectionTags.ai_apps + guide.tracking)"
+                data-external
+            >
+                <span
+                    data-translate="{{:: 'pci_project_guides_header_' + guide.key }}"
+                ></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </div>
+    <oui-message class="mt-2 mb-0" data-type="info" dismissable>
+        <span
+            data-translate="pci_apps_dashboard_banner_user_and_roles_info"
+        ></span>
+        <p class="m-0">
+            <a
```

Comment:
Maybe it is better to use a button instead of using <a> without an href and sref ?

Reply:
Again, sorry, forget to rebase, this code was for an old P, not linked to this one.